docs/osf-trivia-list/dasharo.md

```diff
@@ -154,3 +154,83 @@ Dasharo is not coreboot firmware distribution because we also may imagine
 Dasharo pure EDKII, oreboot or based on any other open-source firmware
 framework. Dasharo typically is based on coreboot and EDKII, plain coreboot is
 not enough in most cases for booting modern computer.
+
+## What value Dasharo provide in comparison to coreboot?
+
+* Dasharo is open-source firmware distribution based on
+  [coreboot](https://coreboot.org) and other open-source firmware frameworks
+  (e.g. [Tianocore EDKII](https://github.com/tianocore/edk2)), you can think
+  about Dasharo and coreboot relation in the same way as you think about
+  Debian/Ubuntu/RedHat and Linux. That means it can provide better
+  cost-effectiveness, security, transparency and customizability than
+  proprietary alternative.
+* There are seven characterestics of Dasharo:
+    - [zero-touch initial deployment](#dasharo-zero-touch-initial-deployment),
+    - [clean and simple code](#dasharo-clean-and-simple-code),
+    - [long-term maintenance](#dasharo-long-term-maintenance),
+    - [professional support](#dasharo-support-package),
+    - [transparent validation](#dasharo-transparent-validation),
+    - [privacy-respecting implementation](#dashro-privacy-respecting-implementation),
+    - [liberty for the owners](#dasharo-liberty-for-the-owners),
+    - [trustworthiness for all](#dasharo-trustworthiness-for-all).
+
+### Dasharo Zero-Touch Initial Deployment
+
+Documentation supported hardware provide information about initial deployment,
+updates and recovery procedures. Developed by Dasharo Team [Dasharo Tools Suite
+(DTS)][dtsos] operating system support users by automating the deployment
```

Comment:
This sentence sounds weird. Why not start it with the subject? We can skip `Developed by Dasharo Team` if it makes it more complex